site/snippets/signers/alchemy/email-signup.ts

```diff
@@ -0,0 +1,6 @@
+import { signer } from "./signer";
+
+signer.authenticate({
+  type: "email",
+  bundle: "USER_EMAIL",
```

Comment:
was this supposed to be `email: "USER_EMAIL"`?

let's also add the `redirectParams` field with a comment that it's optional and what it does